packages/website-router/worker.ts

```diff
@@ -70,6 +70,27 @@ async function handleEvent(event: FetchEvent, sentry: Toucan): Promise<Response>
     return redirect(sentry, event.request.url, to);
   }
 
+  // Remove all www && https && http from the URL
+  if (
+    event.request.url.startsWith('www') ||
```

Comment:
You don't need to consider the protocol (http/https), because CloudFlare ensures that it's always `https`. 
But, you can also use the parse `URL` object, for example:
```
> new URL('https://www.the-guild.dev/graphql/codegen/')
URL {
  href: 'https://www.the-guild.dev/graphql/codegen/',
  origin: 'https://www.the-guild.dev',
  protocol: 'https:',
  username: '',
  password: '',
  host: 'www.the-guild.dev',
  hostname: 'www.the-guild.dev',
  port: '',
  pathname: '/graphql/codegen/',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
>
```

You can just check if `url.host.startsWith('www.')`